.env.example

```diff
@@ -47,6 +47,9 @@ LLM_BACKEND=watsonx
 # Embedding backend, possible values are: ollama, openai, watsonx, bam, caikit
 EMBEDDING_BACKEND=watsonx
 
+# Extraction backend, possible values are: unstructured-opensource, unstructured-api
+EXTRACTION_BACKEND=unstructured-opensource
```

Comment:
maybe `TEXT_EXTRACTION_BACKEND`?

Reply:
I'd rather keep it one word, same as `LLM_BACKEND` or `EMBEDDING_BACKEND`. Besides, the new backends do more than just text extraction.

Reply:
Yes, but this is always a text extraction. When we use extraction for another object, we will need a separate implementation for that.

Reply:
Not exactly, it is a semantic extraction. For WDU we save a serialize text but we could be saving a document similar to the one output by Unstructured.

Reply:
Meaning there is more than just text in what we're extracting.